api.py

```diff
@@ -68,3 +68,16 @@ def get_oyez_brief(resource_id: int):
     if brief := oyez_brief.from_resource_id(resource_id):
         return brief._asdict()
     abort(HTTPStatus.NOT_FOUND)
+
+@app.route('/cases/cluster')
+def get_case_clusters():
+    case_resource_ids = [int(c) for c in request.args.getlist('cases')]
```

Comment:
You shouldn't need to do this (see line 48 for the similarity endpoint analogue), they should be correctly typed as ints when they come in. If I'm wrong, then disregard.
```suggestion
    case_resource_ids = set(request.args.getlist('cases'))
```
and then just passing `case_resource_ids` instead of `set(case_resource_ids)` in lines 80 and 82 should be good.